### ISSUE TITLE
stop at 5-consensus

### ISSUE THREAD

**Issue by sunnycqcn** [marbl/canu#1247]
hello Developer,
Much appreciated for your developing the excellent software. I met a error stopping 5-consensus step. The error file is as following. Could you help me check what it is the error?
Thanks,
Fuyou

```
    > ./ECD.utgStore/partitionedReads.log 2>&1

-- Finished on Sat Feb 16 18:45:33 2019 (34 seconds) with 166523.421 GB free disk space
----------------------------------------
-- Using slow alignment for consensus (iteration '0').
-- Configured 39 contig and 29 unitig consensus jobs.
-- Finished stage 'consensusConfigure', reset canuIteration.
--
--                            (tag)Concurrency
--                     (tag)Threads          |
--            (tag)Memory         |          |
--        (tag)         |         |          |     total usage     algorithm
--        -------  ------  --------   --------  -----------------  -----------------------------
-- Local: cns        3 GB    5 CPUs x   4 jobs    12 GB   20 CPUs  (consensus)
--
--
-- Running jobs.  First attempt out of 2.
----------------------------------------
-- Starting 'cns' concurrent execution on Sat Feb 16 18:45:33 2019 with 166523.419 GB free disk space (68 processes; 4 concurrently)

    cd unitigging/5-consensus
    ./consensus.sh 1 > ./consensus.000001.out 2>&1
    ./consensus.sh 2 > ./consensus.000002.out 2>&1
    ./consensus.sh 3 > ./consensus.000003.out 2>&1
    ./consensus.sh 4 > ./consensus.000004.out 2>&1
    ./consensus.sh 5 > ./consensus.000005.out 2>&1
    ./consensus.sh 6 > ./consensus.000006.out 2>&1
    ./consensus.sh 7 > ./consensus.000007.out 2>&1
    ./consensus.sh 8 > ./consensus.000008.out 2>&1
    ./consensus.sh 9 > ./consensus.000009.out 2>&1
    ./consensus.sh 10 > ./consensus.000010.out 2>&1
    ./consensus.sh 11 > ./consensus.000011.out 2>&1
    ./consensus.sh 12 > ./consensus.000012.out 2>&1
    ./consensus.sh 13 > ./consensus.000013.out 2>&1
    ./consensus.sh 14 > ./consensus.000014.out 2>&1
    ./consensus.sh 15 > ./consensus.000015.out 2>&1
    ./consensus.sh 16 > ./consensus.000016.out 2>&1
    ./consensus.sh 17 > ./consensus.000017.out 2>&1
    ./consensus.sh 18 > ./consensus.000018.out 2>&1
    ./consensus.sh 19 > ./consensus.000019.out 2>&1
    ./consensus.sh 20 > ./consensus.000020.out 2>&1
    ./consensus.sh 21 > ./consensus.000021.out 2>&1
    ./consensus.sh 22 > ./consensus.000022.out 2>&1
    ./consensus.sh 23 > ./consensus.000023.out 2>&1
    ./consensus.sh 24 > ./consensus.000024.out 2>&1
    ./consensus.sh 25 > ./consensus.000025.out 2>&1
    ./consensus.sh 26 > ./consensus.000026.out 2>&1
    ./consensus.sh 27 > ./consensus.000027.out 2>&1
    ./consensus.sh 28 > ./consensus.000028.out 2>&1
    ./consensus.sh 29 > ./consensus.000029.out 2>&1
    ./consensus.sh 30 > ./consensus.000030.out 2>&1
    ./consensus.sh 31 > ./consensus.000031.out 2>&1
    ./consensus.sh 32 > ./consensus.000032.out 2>&1
    ./consensus.sh 33 > ./consensus.000033.out 2>&1
    ./consensus.sh 34 > ./consensus.000034.out 2>&1
    ./consensus.sh 35 > ./consensus.000035.out 2>&1
    ./consensus.sh 36 > ./consensus.000036.out 2>&1
    ./consensus.sh 37 > ./consensus.000037.out 2>&1
    ./consensus.sh 38 > ./consensus.000038.out 2>&1
    ./consensus.sh 39 > ./consensus.000039.out 2>&1
    ./consensus.sh 40 > ./consensus.000040.out 2>&1
    ./consensus.sh 41 > ./consensus.000041.out 2>&1
    ./consensus.sh 42 > ./consensus.000042.out 2>&1
    ./consensus.sh 43 > ./consensus.000043.out 2>&1
    ./consensus.sh 44 > ./consensus.000044.out 2>&1
    ./consensus.sh 45 > ./consensus.000045.out 2>&1
    ./consensus.sh 46 > ./consensus.000046.out 2>&1
    ./consensus.sh 47 > ./consensus.000047.out 2>&1
    ./consensus.sh 48 > ./consensus.000048.out 2>&1
    ./consensus.sh 49 > ./consensus.000049.out 2>&1
    ./consensus.sh 50 > ./consensus.000050.out 2>&1
    ./consensus.sh 51 > ./consensus.000051.out 2>&1
    ./consensus.sh 52 > ./consensus.000052.out 2>&1
    ./consensus.sh 53 > ./consensus.000053.out 2>&1
    ./consensus.sh 54 > ./consensus.000054.out 2>&1
    ./consensus.sh 55 > ./consensus.000055.out 2>&1
    ./consensus.sh 56 > ./consensus.000056.out 2>&1
    ./consensus.sh 57 > ./consensus.000057.out 2>&1
    ./consensus.sh 58 > ./consensus.000058.out 2>&1
    ./consensus.sh 59 > ./consensus.000059.out 2>&1
    ./consensus.sh 60 > ./consensus.000060.out 2>&1
    ./consensus.sh 61 > ./consensus.000061.out 2>&1
    ./consensus.sh 62 > ./consensus.000062.out 2>&1
    ./consensus.sh 63 > ./consensus.000063.out 2>&1
    ./consensus.sh 64 > ./consensus.000064.out 2>&1
    ./consensus.sh 65 > ./consensus.000065.out 2>&1
    ./consensus.sh 66 > ./consensus.000066.out 2>&1
    ./consensus.sh 67 > ./consensus.000067.out 2>&1
    ./consensus.sh 68 > ./consensus.000068.out 2>&1

-- Finished on Sat Feb 16 20:53:19 2019 (7666 seconds) with 166157.157 GB free disk space
----------------------------------------
-- Using slow alignment for consensus (iteration '1').
-- Configured 39 contig and 29 unitig consensus jobs.
--
-- Consensus jobs failed, retry.
--   job ctgcns/0022.cns FAILED.
--
--
-- Running jobs.  Second attempt out of 2.
----------------------------------------
-- Starting 'cns' concurrent execution on Sat Feb 16 20:53:19 2019 with 166157.153 GB free disk space (1 processes; 4 concurrently)

    cd unitigging/5-consensus
    ./consensus.sh 22 > ./consensus.000022.out 2>&1

-- Finished on Sat Feb 16 20:54:00 2019 (41 seconds) with 166149.952 GB free disk space
----------------------------------------
-- Using slow alignment for consensus (iteration '2').
-- Configured 39 contig and 29 unitig consensus jobs.
--
-- Consensus jobs failed, tried 2 times, giving up.
--   job ctgcns/0022.cns FAILED.
--

ABORT:
ABORT: Canu 1.8
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:
```

**Comment by skoren** [comment]
What does the error in the ```unitigging/5-consensus/consensus.000022.out``` report? Have you tried the suggestions in issue #1225, namely increasing the tolerated error rate in consensus.sh?

**Comment by skoren** [comment]
Idle, duplicate of #1225. Consensus changes/fixes will be tracked in #1288
